spec/features/idv/in_person_spec.rb

```diff
@@ -503,6 +503,73 @@
 
         expect(page).to have_current_path(idv_in_person_step_path(step: :address), wait: 10)
       end
+
+      it 'shows hints when user selects Puerto Rico as state',
+         allow_browser_log: true do
+        sign_in_and_2fa_user
+        begin_in_person_proofing
+        complete_location_step
+        complete_prepare_step
+        expect(page).to have_current_path(idv_in_person_step_path(step: :state_id), wait: 10)
+
+        # state id page
+        select 'Puerto Rico',
+               from: t('in_person_proofing.form.state_id.identity_doc_address_state')
+
+        expect(page).to have_content(I18n.t('in_person_proofing.form.state_id.address1_hint'))
+        expect(page).to have_content(I18n.t('in_person_proofing.form.state_id.address2_hint'))
+
+        # change state selection
+        fill_out_state_id_form_ok(double_address_verification: true)
+        expect(page).not_to have_content(I18n.t('in_person_proofing.form.state_id.address1_hint'))
+        expect(page).not_to have_content(I18n.t('in_person_proofing.form.state_id.address2_hint'))
+
+        # re-select puerto rico
+        select 'Puerto Rico',
+               from: t('in_person_proofing.form.state_id.identity_doc_address_state')
+        click_idv_continue
+
+        expect(page).to have_current_path(idv_in_person_step_path(step: :address), wait: 10)
```

Comment:
This test has a lot of `wait`s, which we have been intentionally trying to remove where possible. Can you minimize the usage of `wait` in this test?